boards/arm64/xenvm/Kconfig.defconfig

```diff
@@ -15,4 +15,10 @@ config XEN_INITIAL_DOMAIN
 config BOARD
 	default "xenvm"
 
+config XENVM_USE_GIC_V3
+	bool "Xen VM using GICv3"
+	default n
+	help
+	  Configure Xen VM to use GICv3 instead of default GICv2.
```

Comment:
Because GIC option in Zephyr is invisible symbol. You cannot select it in defconfig. Also the convention in Zephyr is that GIC version is set only by SOC.

Reply:
Uhm, right but I don't particularly like this. What about removing this entirely from the SOC level (from `Kconfig.soc`) and creating two different boards (`BOARD_XENVM_GICV2` and `BOARD_XENVM_GICV3` or similar)?

Reply:
This would have the following consequences:
1. Breaking the SOC->GIC bond. GIC depends on SOC and not on BOARD. Currently in Zephyr, only SOC selects GIC.
So your solution would break that bond imposed by the author of Kconfig.gic(commit 11d0f0a294).
2. Breaking the backwards compatibility. Creating two different boards (including renaming xenvm to xenvm_gicv2) would have consequences for users/companies utilising CI to build -DBOARD=xenvm. xenvm board would not work anymore.
3. Unnecessary increased amount of code. Two different boards would impose creating two directories that do not differ too much.

My change was implemented not to break the backwards compatibility and to be as small and simple as possible.

Reply:
Ok, I will need to polish a bit the GIC story because I don't like the way it currently is. Ok, let's stick to your solution.